exampleCourse/questions/templateFixedMultipleManyAnswers/info.json

```
@@ -0,0 +1,7 @@
+{
+    "uuid": "f24e0597-bb8c-42aa-abef-6a96b6afcca8",
+    "title": "Template pl-multiple-choice with randomly chosen correct answer and distractors",
```

```suggestion
    "title": "Template pl-multiple-choice: Randomly chosen correct answer and distractors",
```

Making the title format consistent with the "Fixed correct answer" title.

---

Argh, old data here. Done in the last commit round.